### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by syleeeee**
Create CONTRIBUTING.md same as other projects. 
- Should we specify the title as "How to contribute to Central Dogma Go project"? 
- There's no label named 'question-answered' at [Issues](https://github.com/line/centraldogma-go/issues), so I used 'question' label to [the list of previously answered questions](https://github.com/line/centraldogma-go/issues?q=label%3Aquestion).

### CODE REVIEW

**Comment by trustin** [comment]
- Title - +1 for updating the title: 'How to contribute to Central Dogma Go client project`
- `question-answered` label has been replaced with `question`. :sweat_smile:

**Comment by syleeeee** [comment]
I'll enable cla-assistant after this P/R be merged.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## How to contribute to Central Dogma Go project
```
Comment (trustin) [inline CONTRIBUTING.md]: How about: Central Dogma Go project -> Go client library for Central Dogma ? :sweat_smile:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-## How to contribute to Central Dogma Go project
+## How to contribute to Go client library for Central Dogma project
 
-First of all, thank you so much for taking your time to contribute! Central Dogma Go is not very different from any other open source projects you are aware of. It will be amazing if you could help us by doing any of the following:
+First of all, thank you so much for taking your time to contribute! Central Dogma Go client library is not very different from any other open source projects you are aware of. It will be amazing if you could help us by doing any of the following:
```
Comment (trustin) [inline CONTRIBUTING.md]: Central Dogma Go client library -> This project

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ First of all, thank you so much for taking your time to contribute! Central Dogm
 ### Contributor license agreement
 
 When you are sending a pull request and it's a non-trivial change beyond fixing typos, please sign [the ICLA (individual contributor license agreement)](https://cla-assistant.io/line/centraldogma-go).  
+Note that this ICLA covers [Central Dogma project](https://github.com/line/centraldogma) and its subproject, which means you can contribute to line/centraldogma and line/centraldogma-go at once by signing this ICLA.
```
Comment (trustin) [inline CONTRIBUTING.md]: - its subproject -> its subprojects
- `line/centraldogma` -> `[line/centraldogma](https://github.com/line/centraldogma)`
- `line/centraldogma-go` -> `[line/centraldogma-go](https://github.com/line/centraldogma-go)`

**Comment by trustin** [comment]
Thanks, @syleeeee !

**Pull request merged**